qcengine/config.py

```diff
@@ -53,7 +53,7 @@ def get_global(key: Optional[str] = None) -> Union[str, Dict[str, Any]]:
             _global_values["cpu_brand"] = _global_values["cpuinfo"]["brand_raw"]
         except KeyError:
             # Remove this if py-cpuinfo is pinned to >=6.0.0
-            _global_values["cpu_brand"] = _global_values["cpuinfo"]["brand"]
+            _global_values["cpu_brand"] = _global_values["cpuinfo"].get("brand", "unknown")
```

Comment:
not a strong suggestion -- just wondering if `(none)` or `(unknown)` would be more consistent with other "missing" schema values (like `model.basis = "(none)"`)

Reply:
I'm all for simplifying the vocabulary. `(unknown)` would be my vote (the CPU has a brand, we just don't know it).